manager/allocator/network.go

```diff
@@ -735,10 +735,10 @@ func (a *Allocator) allocateTask(ctx context.Context, tx store.Tx, t *api.Task)
 	}
 
 	// Update the network allocations and moving to
-	// ALLOCATED state on top of the latest store state.
+	// PENDING state on top of the latest store state.
 	if a.taskAllocateVote(networkVoter, t.ID) {
-		if storeT.Status.State < api.TaskStateAllocated {
-			updateTaskStatus(storeT, api.TaskStateAllocated, "allocated")
+		if storeT.Status.State < api.TaskStatePending {
+			updateTaskStatus(storeT, api.TaskStatePending, "allocated")
```

Comment:
From the existing status messages. I do not find a way to tell which tasks are blocked from scheduler.